packages/ckeditor5-dev-changelog/src/utils/getsectionswithentries.ts

```diff
@@ -76,86 +72,63 @@ function getIssuesLinks( issues: Array<string> | undefined, prefix: string, gitH
 	}
 
 	const links = issues.map( String ).map( issue => {
-		if ( issue.match( sameRepoIssuePattern ) ) {
-			return `[#${ issues }](${ gitHubUrl }/issues/${ issues })`;
+		if ( issue.match( ISSUE_PATTERN ) ) {
+			return `[#${ issue }](${ gitHubUrl }/issues/${ issue })`;
 		}
 
-		const differentRepoMatch = issue.match( differentRepoIssuePattern );
+		const differentRepoMatch = issue.match( ISSUE_SLUG_PATTERN );
 
 		if ( differentRepoMatch ) {
 			const { owner, repository, number } = differentRepoMatch.groups as DifferentRepoIssue;
 
 			return `[${ issue }](https://github.com/${ owner }/${ repository }/issues/${ number })`;
 		}
 
+		const repoUrlMatch = issue.match( ISSUE_URL_PATTERN );
+
+		if ( repoUrlMatch ) {
+			const { owner, repository, number } = repoUrlMatch.groups as DifferentRepoIssue;
+
+			return `[${ owner }/${ repository }#${ number }](${ issue })`;
+		}
+
 		return null;
 	} );
 
-	if ( links.includes( null ) ) {
-		return 'invalid';
-	}
-
 	return `${ prefix } ${ links.join( ', ' ) }.`;
 }
 
-function getSection( {
-	entry,
-	packagesNames,
-	organisationNamespace,
-	singlePackage,
-	closes,
-	see
-}: {
-	entry: ParsedFile;
-	packagesNames: Array<string>;
-	organisationNamespace: string;
-	singlePackage: boolean;
-	closes: string | null;
-	see: string | null;
-} ): SectionName {
-	const packagesNamesNoNamespace = packagesNames.map( packageName => packageName.replace( `${ organisationNamespace }/`, '' ) );
-	const breakingChange = entry.data[ 'breaking-change' ];
-	const type = entry.data.type;
-
-	if ( closes === 'invalid' ) {
+function getSection( { entry, singlePackage, isValid }: { entry: ParsedFile; singlePackage: boolean; isValid: boolean } ): SectionName {
+	if ( !isValid ) {
 		return 'invalid';
 	}
 
-	if ( see === 'invalid' ) {
-		return 'invalid';
-	}
-
-	if ( entry.data.scope && !entry.data.scope.every( scope => packagesNamesNoNamespace.includes( scope ) ) ) {
-		return 'invalid';
-	}
-
-	// If someone tries to use generic breaking change instead of minor/major in monorepo, the entry is invalid.
-	if ( !singlePackage && breakingChange === true ) {
-		return 'invalid';
-	}
+	const breakingChangeNormalized = entry.data.breakingChangeNormalized;
 
 	// If someone tries to use minor/major breaking change in a single package, we simply cast it to a generic breaking change.
-	if ( singlePackage && typeof breakingChange === 'string' ) {
-		return 'breaking';
-	}
-
-	if ( typeof breakingChange === 'string' ) {
-		return breakingChange;
-	}
+	if ( singlePackage ) {
+		if ( breakingChangeNormalized === 'minor' || breakingChangeNormalized === 'major' || breakingChangeNormalized === true ) {
```

Comment:
```suggestion
		if ( [ true, 'minor', 'major' ].includes( breakingChangeNormalized ) ) {
```

Nitpick, feel free to ignore.